coins/ethereum/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ethereum"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+thiserror = "1"
+serde_json = "1.0"
+serde = "1.0"
+hex-literal = "0.3"
+ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["abigen", "ethers-solc"] }
+eyre = "0.6"
+tokio = { version = "1.19", features = ["macros"] }
```

Comment:
Also should not be present here, except as a dev-depend. Sorry for any confusion I contributed on the processor <-> coin relation.